backend/test/conftest.py

```diff
@@ -1,5 +1,26 @@
 import os
 import sys
 
+TEST_DATABASE_URL = "postgresql://devuser:devuser@db:5432/local"
```

Comment:
This should probably be set via the config object. I'm leaving this as is in the interest of reducing a ballooning PR.